### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Tommymarvel**
This is a react to-do list using react hook and destructing method

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+/* 
+const root = ReactDOMClient.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <BrowserRouter>
+      <App />
+    </BrowserRouter>
+  </React.StrictMode>
+);
+*/
```
Comment (degisew) [inline src/index.js]: Kindly, remove dead codes. It makes your codebase unclean.

**Review by degisew — COMMENTED** [review]
Hi @Tommymarvel,

You did a great job so far 👏 

Even if you did a great effort on this, there are some issues that you still need to work on to go to the next project but you are almost there!



## To highlight 💪🏻

- No linter errors ✔️
- Gitflow was correctly used ✔️
- Professional README
- Descriptive PR ✔️ 

# Status: Changes Required  ♻️

 **Check the comment under the review.**

 
## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏 💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (Tommymarvel) [reply src/index.js]: @degisew Thanks for the code review

**Comment by ZikaZaki** [comment]
Great work @Tommymarvel.

**Pull request merged**